pyproject.toml

```diff
@@ -40,10 +41,12 @@ ignore = [
     "TRY",  # no triceratops.
     "NPY",  # remove numpy
     "Q000",  # double quotes
+    "EM101",  # String literals in exception
+    "B028",  # No explicit stack level in warn
+    "FIX002",  # Fixme
+    "TD003",  # Missing issue link on the line following this TODO"
 ]
 
-exclude = ["venv/*", ".venv/*", "build/*", "doc/*", ".github/*"]
```

Comment:
This should be kept, any reason to not exclude these?

Reply:
I was just moving it up to the main section

Reply:
Doubled checked the merge. `exclude` was indeed moved to the ruff main secion.
```toml
[tool.ruff]
exclude = ["venv/*", ".venv/*", "build/*", "doc/*", ".github/*", "tests"]
```